entrypoint.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/sh
+
+set -o errexit
+trap 'call_exit $LINE_NO' EXIT
+
+call_exit()
+{
+echo "at call_exit.."
+echo  "exit code:" $?
+echo "reference: "  $0
+}
+
+# This relies on fact that /tmp is defined as k8s emptyDir volume
+echo '/tmp/core.%h.%e.%t' > /proc/sys/kernel/core_pattern
+ulimit -c unlimited
+
+exec /usr/local/bin/zrepl
```

Comment:
Here, we are executing `zrepl` through exec. So `entrypoint.sh` process image will be replaced by `zrepl` process image and thus `call_exit` will never get executed on.

Reply:
this code was taken from cstor repo. if I understand it right, the purpose of this trap is to print the line number in shell script if we exit prematurely before we get to the last line where zrepl is executed. The script can exit at any line because of the `set -o errexit` option. In that case it prints the line number to give a clue where the entrypoint script has failed.

however when I tested an error path in the script, all it prints is:
```
entrypoint.sh: 14: entrypoint.sh: cannot create /proc/sys/kernel/core_pattern: Permission denied
at call_exit..
exit code: 0
reference:  entrypoint.sh
```
I don't see any value in last three lines which are generated by the trap function. I will remove it. thanks.